src/Parser/Priest/Discipline/Modules/Spells/TwistOfFate.js

```diff
@@ -60,8 +63,32 @@ class TwistOfFate extends Analyzer {
     this.healing += calculateEffectiveHealing(event, TWIST_OF_FATE_HEALING_INCREASE);
   }
 
+  suggestions(when) {
+    if (this.hasSoulOfTheHighPriest === false) {
+      when(this.owner.getPercentageOfTotalHealingDone(this.healing)).isLessThan(SuggestionThresholds.TWIST_OF_FATE_TALENT_PERCENT.minor)
+        .addSuggestion((suggest, actual, recommended) => {
+          return suggest(<span>Consider picking a different talent than <SpellLink id={SPELLS.TWIST_OF_FATE_TALENT.id}/>. Castigation will give a consistent 3-5% increase and Schism provides a significant DPS increase if more healing is not needed.</span>)
+            .icon(SPELLS.TWIST_OF_FATE_TALENT.icon)
+            .actual(`${formatPercentage(actual)}% of total healing`)
+            .recommended(`>${formatPercentage(recommended)}% is recommended.`)
+            .regular(SuggestionThresholds.TWIST_OF_FATE_TALENT_PERCENT.regular)
+            .major(SuggestionThresholds.TWIST_OF_FATE_TALENT_PERCENT.major);
+        });
+    } else {
+      when(this.owner.getPercentageOfTotalHealingDone(this.healing)).isLessThan(SuggestionThresholds.TWIST_OF_FATE_LEGO_PERCENT.minor)
+        .addSuggestion((suggest, actual, recommended) => {
+          return suggest(<span>Consider picking a different legendary other than <ItemLink id={ITEMS.SOUL_OF_THE_HIGH_PRIEST.id}/>. If the burst healing during dangerous situations from Twist of Fate is not needed, you will find greater average healing from several other legendaries.</span>)
+            .icon(SPELLS.TWIST_OF_FATE_TALENT.icon)
+            .actual(`${formatPercentage(actual)}% of total healing`)
+            .recommended(`>${formatPercentage(recommended)}% is recommended.`)
+            .regular(SuggestionThresholds.TWIST_OF_FATE_LEGO_PERCENT.regular)
+            .major(SuggestionThresholds.TWIST_OF_FATE_LEGO_PERCENT.major);
+        });
+    }
+  }
+
   statistic() {
-    if(!this.hasTwistOfFate) { return; }
+    if(!this.active) { return; }
```

Comment:
Nit pick: you should generally return `null` in places where the default is a React node. `undefined` is normally not an accepted value (although I manually handle the `statistic` method to account for it, it's still best practice and might break in the future).